packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -251,12 +278,23 @@ ${JSON.stringify(webhookBody, null, 4)}
         setPluginStatus({ lastFetched: res.body.timestamp })
         requireFullRebuild = true
       } else {
+        fastBuildsSpan.setTag(
+          `sourceNodes.deltaFetch.nodesChanged`,
+          res.body.entities.length
+        )
+
+        const touchNodesSpan = tracer.startSpan(`touchNodes`, {
```

Comment:
touchNodes seems to start taking an inordinate amount of time as the node count grows. So let's track it.